wagtail/images/views/images.py

```diff
@@ -34,19 +34,31 @@
 
 
 class BaseListingView(TemplateView):
+    ORDERING_OPTIONS = {
+        "-created_at": _("Newest"),
+        "created_at": _("Oldest"),
+        "title": _("Title: (A -> Z)"),
+        "-title": _("Title: (Z -> A)"),
+        "file_size": _("File size: (low to high)"),
+        "-file_size": _("File size: (high to low)"),
+    }
+
     @method_decorator(permission_checker.require_any("add", "change", "delete"))
     def get(self, request):
         return super().get(request)
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        # Get images (filtered by user permission)
+        # Ordering
+        ordering = self.request.GET.get("ordering", "-created_at")
```

Comment:
question: what happens if I pass a bogus value for the `ordering` GET param?

Reply:
I believe do something similar in other places, so that works for me

Reply:
It'd be useful to refer to the logic in https://github.com/wagtail/wagtail/blob/main/wagtail/admin/views/generic/models.py#L144-L156, especially as we'll probably be porting this view over to be based on the generic view fairly soon :-)

Reply:
aye, nice one. I was looking for a good example. Nice save @gasman :D

Reply:
@gasman, I've added `get_valid_orderings` and `get_ordering` methods similar to how it's done in the generic view with a comment to remove them when this view will be based on the generic one.